templates/eventCards.handlebars

```diff
@@ -45,7 +45,7 @@
         {{/if}}
         {{#if link}}
         <li class="list-group-item list-item-no-border">
-          <a href="{{link}}">{{#if linkName}}linkName{{/if}}{{#unless linkName}}Find out more{{/unless}}</a>
+          <a href="{{link}}">{{#if linkName}}{{linkName}}{{/if}}{{#unless linkName}}Find out more{{/unless}}</a>
```

Comment:
Hey, so I've fixed this on the dev branch. Can you make sure you've based this from the development branch, and do the pull request into there too? Thanks